### PR TITLE
Undone command

### LINKED ISSUE #178: Do undone command

### PR DESCRIPTION
**Pull request by ZuoZhuolin**
This closes #178

### CODE REVIEW

**Comment by turtle96** [comment]
Testing pass.
Rmb to add tests for next iteration

**Comment by turtle96** [comment]
@CS2103AUG2016-W15-C3/code-quality 
@CS2103AUG2016-W15-C3/documentation 
@CS2103AUG2016-W15-C3/testing 

pls review

**Comment by Mamtha3005** [comment]
Documentation reviewed

File: src/main/java/seedu/taskell/logic/commands/UndoneCommand.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        if (lastShownList.size() < targetIndex) {
+            indicateAttemptToExecuteIncorrectCommand();
+            return new CommandResult(Messages.MESSAGE_INVALID_TASK_DISPLAYED_INDEX);
+        }
+        
+        TaskStatus incompleteStatus = new TaskStatus(TaskStatus.INCOMPLETE);
+        
+        ReadOnlyTask taskToBeUndone = lastShownList.get(targetIndex - 1);
+        Task newTask = null;
+        if(taskToBeUndone.getTaskStatus().taskStatus().equals(TaskStatus.INCOMPLETE)){
```
Comment (marcusngwenjian) [inline src/main/java/seedu/taskell/logic/commands/UndoneCommand.java:47]: whitespace is missing between if and the brackets

**Comment by marcusngwenjian** [comment]
Make the necessary changes for code quality.
Refer to issue #183 for follow-up action.

**Pull request merged**